speed:
  angular_scale:  1.0           # Scale factor for angular velocity
  linear_scale:   1.0           # Scale factor for linear velocity

control_method: "velocity"      # Velociy or torque control
can_interface:  "vcan0"          # Name of the CAN network interface
wheel_ids:      [1, 2, 3, 4]    # fr, fl, rr, rl
loop_rate:      20              # Control loop frequency